test_unstructured_inference/test_api.py

```diff
@@ -55,6 +57,68 @@ def test_bad_route_404():
     assert response.status_code == 404
 
 
+def test_layout_v02_api_parsing_image():
+
+    filename = os.path.join("sample-docs", "test-image.jpg")
+
+    client = TestClient(app)
+    response = client.post(
+        "/layout_v1/image",
+        headers={"Accept": "multipart/mixed"},
+        files=[("files", (filename, open(filename, "rb"), "image/png"))],
+    )
+    doc_layout = jsons.load(response.json(), DocumentLayout)
+    assert len(doc_layout.pages) == 1
+    # The example sent to the test contains 13 detections
```

Comment:
Please add `# NOTE(benjamin) -` to comments like these so future us knows who to ask if there are questions :)